plugin.py

```diff
@@ -608,6 +618,42 @@ def onHeartbeat(self):
         else:
             self.mqttClient.Ping()
 
+        # Sensor support
+        # Timing out sensors
+        #Domoticz.Debug( "OnHeartbeat: Settings " + str( Settings ) )
```

Comment:
Don't leave commented out code here, make the logging conditional instead
```py
if DEBUG_SENSORS:
    Domoticz.Debug( "OnHeartbeat: Settings " + str( Settings ) )
```

Reply:
If I do it like this, the if statement will be executed continuously, and wasting CPU time, isn't it ? 
I'm OK for removing this line completely, if you do not like the #

Reply:
> the if statement will be executed continuously, and wasting CPU time

The waste of CPU time for a few conditional statements is very small, either hide the extra prints behind a condition or if it's not critical just remove them.